apps/browser/src/tools/popup/send-v2/send-v2.component.html

```diff
@@ -6,11 +6,11 @@
       <app-current-account></app-current-account>
     </ng-container>
   </popup-header>
-  <div slot="above-scroll-area" class="tw-p-4">
+  <div slot="above-scroll-area" [ngClass]="{ 'tw-p-4': (sends$ | async)?.length }">
     <bit-callout *ngIf="sendsDisabled" [title]="'sendDisabled' | i18n">
       {{ "sendDisabledWarning" | i18n }}
     </bit-callout>
-    <ng-container *ngIf="!sendsDisabled">
+    <ng-container *ngIf="!sendsDisabled && (sends$ | async)?.length">
```

Comment:
As mentioned before I'd expect as a user with Sends created before the policy is enabled to be able to filter and sort their existing Sends, even if they can only view and delete them.

If you remove the `!sendsDisabled` you'll notice that relying on `sends$` doesn't work well, as it is also used to show filtered results. This then completely removes the search and filter capabilities. That why I suggest:
```suggestion
    <ng-container *ngIf="listState !== sendState.Empty">
```